crates/ink/codegen/src/generator/trait_def/trait_registry.rs

```diff
@@ -206,6 +206,7 @@ impl TraitRegistry<'_> {
             }
         };
         quote_spanned!(span=>
+            #( #attrs )*
```

Comment:
Maybe you want to use `cfg_attrs` here because it is generated part of the trait(not expected by the user)?